File: AddressSearchSample/Objective-C/AddressSearchSample/AddressSearchKeywordViewController.m
//
//  AddressSearchKeywordViewController.m
//  AddressSearchSample
//
//  Copyright © 2559 Globtech. All rights reserved.
//

#import "AddressSearchKeywordViewController.h"

@implementation AddressSearchKeywordViewController

- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    if ([segue.identifier isEqualToString:@"resultSegue"]) {
        AddressSearchResultViewController *addressSearchResultViewController = segue.destinationViewController;
        NTAddressSearchParameter *param = [[NTAddressSearchParameter alloc] initWithKeyword:_txtKeyword.text];
        addressSearchResultViewController.param = param;
    }
}

@end

File: AddressSearchSample/Swift/AddressSearchSample/AddressSearchKeywordViewController.swift
//
//  AddressSearchKeywordViewController.swift
//  AddressSearchSample
//
//  Copyright © 2559 Globtech. All rights reserved.
//

import UIKit
import NOSTRASDK
class AddressSearchKeywordViewController: UIViewController {

    @IBOutlet weak var txtKeyword: UITextField!
    
    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
        if segue.identifier == "resultSegue" {
            let resultViewController = segue.destinationViewController as? AddressSearchResultViewController;
            
            let param = NTAddressSearchParameter(keyword: txtKeyword.text!);
            
            resultViewController?.addressSearchParam = param;
            
        }
    }
    

}
